TestRange.c

```diff
@@ -0,0 +1,18 @@
+#include <assert.h>
+#include "RangeCapture.h"
+
+int main()
+{
+  int CurrentSamples[] = {3,3,5,4,10,11,12};
+  int CurrentSamplesSize = sizeof(CurrentSamples) / sizeof(CurrentSamples[0]);
+  
+  int *CheckRange;
+  
+  CheckRange = DetectRange(CurrentSamples,CurrentSamplesSize);
+  printf("CheckRange = %d\n",CheckRange);
+  assert(*(CheckRange+4)) == 3);
```

Comment:
if range count is the 3rd element in each 1d array , then why is 4 added? doesnt this access the fifth element?